server/config/backup.php

```diff
@@ -0,0 +1,281 @@
+
+            /*
+             * If this is an empty string, the name field on the webhook will be used.
+             */
+            'username' => '',
+
+            /*
+             * If this is an empty string, the avatar on the webhook will be used.
+             */
+            'avatar_url' => '',
+        ],
+    ],
+
+    /*
+     * Here you can specify which backups should be monitored.
+     * If a backup does not meet the specified requirements the
+     * UnHealthyBackupWasFound event will be fired.
+     */
+    'monitor_backups' => [
+        [
+            'name' => env('APP_NAME', 'laravel-backup'),
+            'disks' => ['local'],
+            'health_checks' => [
+                \Spatie\Backup\Tasks\Monitor\HealthChecks\MaximumAgeInDays::class => 1,
+                \Spatie\Backup\Tasks\Monitor\HealthChecks\MaximumStorageInMegabytes::class => 5000,
+            ],
+        ],
+
+        /*
+        [
+            'name' => 'name of the second app',
+            'disks' => ['local', 's3'],
+            'health_checks' => [
+                \Spatie\Backup\Tasks\Monitor\HealthChecks\MaximumAgeInDays::class => 1,
+                \Spatie\Backup\Tasks\Monitor\HealthChecks\MaximumStorageInMegabytes::class => 5000,
+            ],
+        ],
+        */
+    ],
+
+    'cleanup' => [
+        /*
+         * The strategy that will be used to cleanup old backups. The default strategy
+         * will keep all backups for a certain amount of days. After that period only
+         * a daily backup will be kept. After that period only weekly backups will
+         * be kept and so on.
+         *
+         * No matter how you configure it the default strategy will never
+         * delete the newest backup.
+         */
+        'strategy' => \Spatie\Backup\Tasks\Cleanup\Strategies\DefaultStrategy::class,
+
+        'default_strategy' => [
+
+            /*
+             * The number of days for which backups must be kept.
+             */
+            'keep_all_backups_for_days' => 1000,
+
+            /*
+             * The number of days for which daily backups must be kept.
+             */
+            'keep_daily_backups_for_days' => 1000,
+
+            /*
+             * The number of weeks for which one weekly backup must be kept.
+             */
+            'keep_weekly_backups_for_weeks' => 1000,
+
+            /*
+             * The number of months for which one monthly backup must be kept.
+             */
+            'keep_monthly_backups_for_months' => 1000,
+
+            /*
+             * The number of years for which one yearly backup must be kept.
+             */
+            'keep_yearly_backups_for_years' => 1000,
+
+            /*
+             * After cleaning up the backups remove the oldest backup until
+             * this amount of megabytes has been reached.
+             */
+            'delete_oldest_backups_when_using_more_megabytes_than' => 50000,
+        ],
+    ],
```

Comment:
Maybe reduce this to some lower sensible level, eg. we keep all backups from last 3 days, but then only 1 weekly/monthly backup?